packages/popover/src/Popover.stories.tsx

```diff
@@ -111,23 +118,28 @@ const meta: StoryMetaType<typeof Popover> = {
       },
       // eslint-disable-next-line react/display-name
       decorator: Instance => {
+        // eslint-disable-next-line react-hooks/rules-of-hooks
+        const [active, setActive] = useState<boolean>(false);
+
         return (
           <div
             className={css`
               position: relative;
-              width: 50vw;
+              width: 60vw;
               height: 150px;
               display: flex;
               align-items: center;
               justify-content: center;
             `}
           >
-            <Button>
+            <Button onClick={() => setActive(active => !active)}>
               Button Text
               <Instance
+                buttonText={undefined}
                 className={css`
                   background-color: ${color.light.background.primary.default};
                 `}
+                dismissMode="manual"
```

Comment:
I don't remember what the answer was last time: does this need to be _inside_ the Button?